modules/hostap/Kconfig

```diff
@@ -24,11 +24,8 @@ config WIFI_NM_WPA_SUPPLICANT
 if WIFI_NM_WPA_SUPPLICANT
 
 config HEAP_MEM_POOL_ADD_SIZE_HOSTAP
-	def_int 85000 if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE && !MBEDTLS_ENABLE_HEAP
-	def_int 85000 if WIFI_NM_HOSTAPD_AP
-	def_int 40000 if WIFI_NM_WPA_SUPPLICANT_AP
-	# 8192 for MbedTLS heap
-	def_int 21808 if MBEDTLS_ENABLE_HEAP
+	def_int 66560 if WIFI_NM_HOSTAPD_AP
+	def_int 41808 if WIFI_NM_WPA_SUPPLICANT_AP || WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
 	# 30K is mandatory, but might need more for long duration use cases
 	def_int 30000
```

Comment:
is this still the same? don't we expect it to be decreased?

Reply:
already reduce the NET_SOCKETPAIR_BUFFER_SIZE

Reply:
no, I meant, as `NET_SOCKETPAIR_BUFFER_SIZE` is reduced now, should we also adjust/reduce `30000` ?

Reply:
`30000` is no need to change, it's for basic usage